### PR TITLE
feat: Adding docs on Pipelines installation

### PR DESCRIPTION
**Pull request by yhakbar**

### CODE REVIEW

File: docs/2.0/docs/pipelines/installation/overview.md [hunk 1]
```diff
@@ -1,3 +1,55 @@
 # Pipelines Installation Overview
 
-This page is under construction.
\ No newline at end of file
+Given that Pipelines is multiple different technologies working in concert to deliver a comprehensive CI/CD solution, there are a number of different ways to achieve installation of Pipelines.
```
Comment (oredavids) [inline docs/2.0/docs/pipelines/installation/overview.md:3]:
```suggestion
Given that Pipelines is a collection of multiple technologies working in concert to deliver a comprehensive CI/CD solution, there are a number of different ways to achieve installation of Pipelines.
```

File: docs/2.0/docs/pipelines/installation/overview.md [hunk 2]
```diff
@@ -1,3 +1,55 @@
...
+
+Customers that have setup DevOps Foundations have a white glove setup process that automatically results in the full installation of Pipelines as a side effect of the setup process.
+
+A Gruntwork solutions engineer will walk you through the process of:
+
+1. Creating a new `infrastructure-live-root` repository from the [`infrastructure-live-root-template`](https://github.com/gruntwork-io/infrastructure-live-root-template) template.
+2. Installation of the [Gruntwork.io GitHub App](https://github.com/apps/gruntwork-io) on the `infrastructure-live-root` repository, or on the organization as a whole. For more information on this, click [here](/docs/pipelines/installation/viagithubapp).
+3. Running the associated `Infrastructure Live Root Bootstrap` workflow to customize the generated repository so that it can be used to deploy infrastructure.
+
+The outcome of following these steps is that you will have a GitHub repository that will continuously deploy infrastructure changes in a GitOps manner without any further manual intervention.
```
Comment (oredavids) [inline docs/2.0/docs/pipelines/installation/overview.md:17]:
```suggestion
The outcome of following these steps is that you will have a GitHub repository that can deploy infrastructure changes in a GitOps manner without any further manual intervention.
```

**Review by oredavids — APPROVED** [review]

File: docs/2.0/docs/pipelines/installation/overview.md [hunk 3]
```diff
@@ -1,3 +1,55 @@
...
+
+## Installation via manual setup
+
+If you are not using DevOps Foundations, or need Pipelines for a standalone repository that already has Terragrunt configurations, you can still install Pipelines directly as a standalone GitHub Actions workflow.
+
+To learn how to do this, navigate to the dedicated documentation for [Adding Pipelines to an Existing Repository](/2.0/docs/pipelines/installation/addingexistingrepo).
+
+## Local Installation
```
Comment (Resonance1584) [inline docs/2.0/docs/pipelines/installation/overview.md:25]: I don't think we should promote this in any way

File: docs/2.0/docs/pipelines/installation/overview.md [hunk 4]
```diff
@@ -1,3 +1,55 @@
...
+
+Most customers have no need to install Pipelines locally. The product has been designed to be installed in CI/CD workflows, and has a lot of functionality that is only relevant in that context.
+
+As a Gruntwork customer, however, you do have the ability to install Pipelines locally if you wish. This can be useful for troubleshooting unexpected behavior in your CI/CD workflows, or for validating your understanding of Pipelines behavior with shorter feedback loops.
+
+To install Pipelines locally, navigate to the [latest GitHub release on the `pipelines-cli` repository](https://github.com/gruntwork-io/pipelines-cli/releases/latest) and download the appropriate binary for your operating system.
+
```
[the remark this answers is not in the document]
Comment (yhakbar) [reply docs/2.0/docs/pipelines/installation/overview.md:32]: I'm not sure what info you're talking about.

File: docs/2.0/docs/pipelines/installation/overview.md (hunk 3 above)
Comment (yhakbar) [reply docs/2.0/docs/pipelines/installation/overview.md]: Tossed.

File: docs/2.0/docs/pipelines/installation/overview.md (hunk 2 above)
Comment (yhakbar) [reply docs/2.0/docs/pipelines/installation/overview.md]: I compromised. LMK if you're ok with it now.

File: docs/2.0/docs/pipelines/installation/overview.md (hunk 1 above)
Comment (yhakbar) [reply docs/2.0/docs/pipelines/installation/overview.md]: Updated.

File: docs/2.0/docs/pipelines/installation/overview.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (oredavids) [reply docs/2.0/docs/pipelines/installation/overview.md]: The comment was for a local install instruction which appears to have been removed.

**Review by oredavids — APPROVED** [review]

**Pull request merged**